kubemarine/templates/plugins/calico-v3.21.yaml.j2

```diff
@@ -4109,6 +4109,10 @@ spec:
         # Mark the pod as a critical add-on for rescheduling.
         - key: CriticalAddonsOnly
           operator: Exists
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
+        - key: node-role.kubernetes.io/control-plane
+          effect: NoSchedule
```

Comment:
Maybe we should make it configurable? Also, does it make sense to tolerate `NoExecute` by default, so that typha could run on `NotReady` workers? For reference:
1. https://kubernetes.io/docs/concepts/scheduling-eviction/_print/#taint-based-evictions
2. `NoExecute` on calico https://github.com/Netcracker/KubeMarine/blob/main/kubemarine/templates/plugins/calico-v3.24.yaml.j2#L4538